packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -68,6 +70,12 @@
     }
   }
 
+  .time-input-hour[disabled],
+  .time-input-minute[disabled],
+  .meridiem-display[disabled] {
+    opacity: 1;
```

Comment:
@amichaelparker - this actually doesn't work without it chained & nested below the .time-input class, otherwise we will need to refactor. For the mobile view see twelve-hour/mobile test and reduce your screen width below 1024px, you'll see this is when we want the original opacity to not be overridden.

Reply:
Understood! Merged it back into .time-input.